crawler/plugins_manager.py

```diff
@@ -37,22 +38,69 @@ def get_plugins(
     return pm.getAllPlugins()
 
 
-def get_emitter_plugins(urls=['stdout://'],
-                        format='csv',
-                        plugin_places=['plugins']):
+def get_emitter_plugin_args(plugin, config):
+    plugin_args = {}
+    if plugin.name in config['emitters']:
+        plugin_args = config['emitters'][plugin.name]
+    return plugin_args
+
+
+def load_emitter_plugins(urls=['stdout://'],
+                         format='csv',
+                         plugin_places=['plugins']):
     category_filter = {"emitter": IEmitter}
+
+    # getting all emitter plugins from crawelr/plugins/emitters/*
     all_emitter_plugins = get_plugins(category_filter, plugin_places)
-    selected_emitter_plugins = []
-    for url in urls:
-        parsed = urlparse.urlparse(url)
-        proto = parsed.scheme
-        for plugin in all_emitter_plugins:
-            plugin_obj = plugin.plugin_object
+
+    # getting enabled emitter pluggins from crawler.conf file
+    conf_enabled_plugins = []
+    config = config_parser.get_config()
+    if 'enabled_emitter_plugins' in config['general']:
+        conf_enabled_plugins = config['general']['enabled_emitter_plugins']
+        if 'ALL' in conf_enabled_plugins:
+            conf_enabled_plugins = [p for p in config['emitters']]
```

Comment:
No, --url from the CLI works as before. This is for the case where the plugins are also enabled in conf file.